### DISCUSSION TITLE
medvram?

### DISCUSSION THREAD

**Discussion by SilentPrayeCG** [Comfy-Org/ComfyUI#1156] [Q&A] [answered]
Hi. Is it possible to boot this Web UI with medvram? Or that arg was Automatic1111 specific?
And how exactly you even start it with arguments?

ps I've checked "similar discussion" button, and it's apparently can be ran with medvram, then question how?

update:
ok, never mind, args just goes at the end of line that run main py script, in start up bat file.

**Comment by comfyanonymous** [comment, marked as answer]
That's the default. There's these if you want it to use more vram: --gpu-only --highvram

Or --lowvram if you want it to use less.

**Reply by ntdviet** [reply]
@comfyanonymous: and where do you put these arguments pls? In your colab notebook, I tried to find a spot where it would fit, but can't find any

**Reply by SilentPrayeCG** [reply]
So it's by default running with medvram? Good. Because, I have 1070, and it chokes with new SDXL models, I found that in Automatic1111 medvram become very handy for those new models.
Tho, still, where do I put arguments, to start ComfyUI with it?

**Reply by DrGrungle** [reply]
Hi there,
I'm still pretty new, and am not familiar with Colab, but...
Wherever you are running the "main.py", whether that be in a .bat file, .sh file, or in the command line, you can just add the --lowvram option straight after main.py, eg:
python3 ./main.py --lowvram --auto-launch
and any other arguments you want to add.
Generation times are longer, but I'm running SDXL 1.0 with a 1650 4GB GTX, i7 7700k and 44GB RAM without any issues on --lowvram.

**Reply by ntdviet** [reply]
Thanks a bunch @DrGrungle. It's funny I've looked into main.py to search for anything "RAM" but of course there isn't :))
Good to know it's just an option in the .py call
On my Colab, it's detecting VRAM > RAM and automatically invoke --highvram, which then runs out pretty damn quickly with SDXL. I need this --medvram. Thanks again

**Reply by Sadpanda07** [reply]
@comfyanonymous where to type this commands (--highvram)?I want to enable highvram mode

**Comment by ghostsquad** [comment]
Any guidance for when to use each option? I have a 3060 12GB.
